app/controllers/presentations_controller.rb

```diff
@@ -11,6 +11,7 @@ def show
 
   def new
     @presentation = Presentation.new
+    @event = Event.find(params[:event_id]) if params[:event_id]
```

Comment:
Actually it 500's. If we did a `.find_by(id: params[:event_id])` that would work but I don't want to set an instance variable to `nil` if I don't have to.